data/lib/core/actionids.lua

```diff
@@ -0,0 +1,7 @@
+aId = {
```

Comment:
this can be `actionIds`, no need to shorten in like that, it's a global variable so short names can introduce naming collisions.